web/src/utilities/utilities.js

```diff
@@ -583,8 +576,11 @@ export const Utilities = {
   },
 
   isLoggedIn() {
-    const hasToken = this.getToken();
-    return !!hasToken;
+    if (localStorage.getItem("isLoggedIn") === "true") {
```

Comment:
No longer store JWT in localStorage. We set isLoggedIn in localStorage when user successfully logs in. https://github.com/replicatedhq/kots/blob/28fef60f1e0f0589b5925f0caf945f06cecd4da4/web/src/features/Auth/components/SecureAdminConsole.tsx#L55